chore: link the new section for using certificates in ms (#1065)
kedacore/keda-docs: content/docs/2.10/operate/cluster.md

## Certificates used by KEDA Metrics Server

> This information has been moved to [security section](./security#use-your-own-tls-certificates)

Reviewer: ```suggestion
To learn more please refer to [security section](./security#use-your-own-tls-certificates)
```
